maxim/logger/anthropic/utils.py

```
@@ -45,47 +79,94 @@ def parse_message_param(
+
+        Raises:
+            ValueError: If the stream list is empty.
+
+        Note:
+            - Text content is extracted from content_block_delta events
+            - Token usage is extracted from message_start events
+            - The response format mimics OpenAI's API for compatibility
+            - Finish reason is set to "stop" by default (Anthropic doesn't provide this directly)
+        """
         if not stream:
             raise ValueError("No response chunks")
```

Ruff (0.11.9)

Avoid specifying long messages outside the exception class

(TRY003)